docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+different aspects of Jupyter software and the community. A central council,
+the Jupyter Executive Council (LINK), makes decisions about project-wide
+goals and policies, while different subprojects handle the actual development
+of the various software components.
+
+Some subprojects take care of broader topics, such as the [Accessibility](),
+[Security](), [Community](), and [Documentation]() projects.
+
+# A (Partial) Tour of the Jupyter Universe
+
+In the following sections, we are going to look at some popular components of
+the Jupyter ecosystem. This is *not* a comprehensive reference of every aspect
+of Jupyter, but rather a big-picture summary that should help illustrate some
+important parts that were discussed earlier.
+
+If you are completely new to *Jupyter*, the project's [About][] page is good
+reading that will introduce you to many of the different subprojects.
+
+If you want to know more about the organizational structure, check out the
+[Governance]() pages.
+
+And if you would like to contribute to the project, have a look at the
+[Get Involved]() page.
+
+The graph below presents the best known software components of the Project.
+
+Connections and groups in this diagram are not formal relationships, but
+simple indicators to help us draw the big picture in the next sections.
+
+```mermaid
+graph TD
+    IPython
+        click IPython href "https://ipython.org" _blank
+
+    subgraph source/code
+        Nbformat[["Notebook file format (.ipynb)"]]
+            click Nbformat href "https://nbformat.readthedocs.io" _blank
+
+        JupyterWidgets>Jupyter-Widgets]
+            click JupyterWidgets href "http://ipywidgets.readthedocs.io" _blank
+    end
+
+    subgraph frontends
+        direction LR
+
+        JupyterLab(Jupyter-Lab)
+            click JupyterLab href "https://jupyterlab.readthedocs.io" _blank
+
+        JupyterNotebook(Jupyter-Notebook)
+            click JupyterNotebook href "https://jupyter-notebook.readthedocs.io" _blank
+    end
+
+    subgraph viewers
+        direction LR
+
+        Binder(Binder)
+            click Binder href "https://mybinder.org" _blank
+
+        Voila(Voilà)
+            click Voila href "https://voila.readthedocs.io" _blank
+
+        Nbviewer[nbviewer]
+            click Nbviewer href "https://nbviewer.org" _blank
+
+        Nbconvert[nbconvert]
+            click Nbconvert href "https://nbconvert.readthedocs.io" _blank
+    end
+
+    JupyterHub(Jupyter-Hub)
+        click JupyterHub href "https://jupyterhub.readthedocs.io" _blank
+
+    JupyterHub --> frontends
+    IPython ~~~ source/code
+    frontends --> source/code
+    viewers --> source/code
+```
+
+### Notebook Editor Programs
+
+- [JupyterLab](https://jupyterlab.readthedocs.io/en/latest/)
+- [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/)
+
+The two most popular notebook editor programs are Jupyter-Notebook and
+JupyterLab, though there are others. Which one you choose to use is
```

Comment:
Rather than describing these as "the two most popular notebook editors", we might want to instead characterize them as two notebook editors **published by Project Jupyter**. There are many other editors produced and published by 3rd-party commercial vendors, including my own employer, and which may be popular. We can't endorse any of them in this document, but I am concerned that this phrasing seems to minimize the outside developer community for notebook editors.

Reply:
I agree the third parts editor are important and would like to emphasize that here. I'll refactor this and add those in.

Reply:
Resolved in 300b0ffd4